opensoundscape/audiomoth.py

```diff
@@ -159,13 +163,17 @@ def _parse_audiomoth_battery_info(comment):
     ...battery state was 4.7V.
     ...battery state was less than 2.5V
     ...battery state was 3.5V and temperature....
+    ...battery was 4.6V and...
 
     Args:
         comment: the full comment string from an audiomoth metadata Comment field
     Returns:
         float of voltage or string describing voltage, eg "less than 2.5V"
     """
-    battery_str = comment.split("battery state was ")[1].split("V")[0] + "V"
+    if "state " in comment:
```

Comment:
Mentioning one more thing discussed earlier. Would be good to change `if "state "` to `if "battery state"` to make it explicit and void changes if another "state" pops up in comments.